libr/anal/p/anal_arm_cs.c

```diff
@@ -3772,9 +3759,9 @@ static void op_fillval (RAnalOp *op, csh handle, cs_insn *insn, int bits) {
 		}
 #endif
 		for (j = 0; j < 3; j++, i++) {
-			set_src_dst (op->src[j], &handle, insn, i, bits);
+			set_src_dst (op->src[j], anal->reg, &handle, insn, i, bits);
```

Comment:
@GustavoLCR Is reg_num/x actually used? Shouldn't it be `j` here instead of `i`?

Reply:
@karliss reg_num/x is used to get the operand inside the instruction, and its `i`, because it can at an offset from `j`